client/fed_test.go

```diff
@@ -407,3 +407,87 @@ func TestGetPublicRead(t *testing.T) {
 		}
 	})
 }
+
+// A test that tests the statHttp function
+func TestStatHttp(t *testing.T) {
+	ctx, _, _ := test_utils.TestContext(context.Background(), t)
+	viper.Reset()
+	common.ResetOriginExports()
+
+	fed := fed_test_utils.NewFedTest(t, bothPublicOriginCfg)
+
+	t.Run("testStatHttpPelicanScheme", func(t *testing.T) {
+		testFileContent := "test file content"
+		// Drop the testFileContent into the origin directory
+		tempFile, err := os.Create(filepath.Join(((*fed.Exports)[0]).StoragePrefix, "test.txt"))
+		assert.NoError(t, err, "Error creating temp file")
+		defer os.Remove(tempFile.Name())
```

Comment:
I saw that `NewFedTest` removes all files at the end of the test, so this is not needed.